src/index.html

```diff
@@ -22,8 +22,28 @@
     <script src="../node_modules/cornerstone-wado-image-loader/dist/cornerstoneWADOImageLoader.min.js"></script>
     <link href="node_modules/@angular/material/prebuilt-themes/indigo-pink.css" rel="stylesheet">
 
+    <!-- Google API Library -->
+    <script src="https://apis.google.com/js/api.js"></script>
+
 </head>
 <body style="background: #f8f9fa">
+
+        <script type="text/javascript">
```

Comment:
IMO się nie ma się co przejmować jakością kodu. Jak działa to git xD
Sam robiłem kod który potem został w metodzie 'doSomethingWojtekKnowsWhat'